### PR TITLE
[kueuectl] Add resourceflavor to pass-through commands.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
Add resourceflavor to pass-through commands.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
CLI: Added resourceflavor to pass-through commands.
```

### CODE REVIEW

File: site/content/en/docs/reference/kubectl-kueue/commands/passthrough.md [hunk 1]
```diff
@@ -27,20 +27,21 @@ kubectl kueue delete cq my-cluster-queue
...
-| edit       |  Edit a resource on the server
-| patch      |  Update fields of a resource
-| delete     |  Delete resources by file names, stdin, resources and names, or by resources and label selector
+| Name       | Description                                                                                    |
+|------------|------------------------------------------------------------------------------------------------|
+| get        | Display one or many resources                                                                  |
+| describe   | Show details of a specific resource or group of resources                                      |
+| edit       | Edit a resource on the server                                                                  |                                                                 
+| patch      | Update fields of a resource                                                                    |                                                                
+| delete     | Delete resources by file names, stdin, resources and names, or by resources and label selector |
```
Comment (trasc) [inline site/content/en/docs/reference/kubectl-kueue/commands/passthrough.md:36]:
```suggestion
| edit       | Edit a resource on the server                                                                  |  
| patch      | Update fields of a resource                                                                    |     
| delete     | Delete resources by file names, stdin, resources and names, or by resources and label selector |
```

File: test/integration/kueuectl/suite_test.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ var (
 	webhookPath = filepath.Join("..", "..", "..", "config", "components", "webhook")
 
 	kueuectlPath   string
-	kassetsPath    string
+	kAssetsPath    string
```
Comment (trasc) [inline test/integration/kueuectl/suite_test.go:52]: why?

Comment (mbobrovskyi) [reply test/integration/kueuectl/suite_test.go]: Just warning of IDE. Reverted.

File: site/content/en/docs/reference/kubectl-kueue/commands/passthrough.md (hunk 1 above)
Comment (mbobrovskyi) [reply site/content/en/docs/reference/kubectl-kueue/commands/passthrough.md]: Fixed

**Comment by alculquicondor** [comment]
/approve
for KEP update
/hold for review from @trasc

**Review by trasc — COMMENTED** [review]
/lgtm

**Pull request merged**